.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
     - NPROC_MAX=8 BUILDTEST_MCU_GROUP=cortex_m4
 
 before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - check_gh_label "Ready for CI build" || exit 1
```

Comment:
Are these lines run in the same environment? I would have imagined Travis launching a new shell instance for each line like `make` does.
Anyway, if it works, it works, otherwise you have to change it into something like `source ./dist/tools/pr_check/check_labels.sh && check_gh_label "Ready for CI build" || exit 1`

Regarding the comment by @BytesGalore, you can use `[ "$BUILDTEST_MCU_GROUP == "static-tests" ] || source blah && check_label ...`